pip/commands/install.py

```diff
@@ -266,6 +276,7 @@ def run(self, options, args):
                     "continue."
                 )
             install_options.append('--home=' + temp_target_dir)
+            install_options.append('--prefix=')
```

Comment:
I added the same thing as already exists for the user command (line 260).
An empty `prefix` is set in order to overwrite one set as a default somewhere else (e.g. as it is the case with homebrew python).